Make docs (#30)
radio-astro-tools/uvcombine: docs/feathering_cube.rst

.. _feathercubes:

Feathering two cubes
====================

.. note:: In progress...

> .. warning:: The method desribed here is currently not tested. Check your results carefully when using.

Reviewer: ```suggestion
.. warning:: The method desribed here is currently not tested as part of the continuous integration suite. Check your results carefully when using.
```

Two cubes can currently be combined with the `~uvcombine.fourier_combine_cubes`.

> This method is appropriate for two cube, each with a single beam size (i.e., the beam

Reviewer: ```suggestion
This method is appropriate for two cubes that each have a single beam size (i.e., the beam
```

> does not change between spectal channels). The low resolution beam size is also set

Reviewer: ```suggestion
does not change between spectral channels). The low resolution beam size is also set
```
